service-discovery/kubernetes/src/main/java/io/smallrye/stork/servicediscovery/kubernetes/KubernetesServiceDiscoveryProvider.java

```diff
@@ -18,6 +18,7 @@
 @ServiceDiscoveryAttribute(name = "k8s-host", description = "The Kubernetes API host.")
 @ServiceDiscoveryAttribute(name = "k8s-namespace", description = "The namespace of the service. Use all to discover all namespaces.")
 @ServiceDiscoveryAttribute(name = "application", description = "The Kubernetes application Id; if not defined Stork service name will be used.")
+@ServiceDiscoveryAttribute(name = "port-name", description = "The Kubernetes application port name; if not defined and there is only one port defined, Stork will use it; otherwise it will use the port 80.")
```

Comment:
Where in the code are we taking the 80 port?

Reply:
I would refactor the 'otherwise' part of this to explain that this param is used to select the port when multiple ones are exposed, this is important and is not being mentioned.

Reply:
> Where in the code are we taking the 80 port?

I guess using the fallback service discovery: https://github.com/smallrye/smallrye-stork/blob/72b13e399293ee971079be9f8212807794cf1400/service-discovery/static-list/src/main/java/io/smallrye/stork/servicediscovery/staticlist/StaticListServiceDiscoveryProvider.java#L42, but I can't tell for sure.

> I would refactor the 'otherwise' part of this to explain that this param is used to select the port when multiple ones are exposed, this is important and is not being mentioned.

Will update it.

Reply:
Yes, I know it's the static service discovery, though it won't use the container port, but 80 (I can't tell where in the code, 80 is selected, I haven't played much with stork). What is happening is that the port is 0 and somehow then it uses 80.

Reply:
Mmm :thinking:. Given that we don't see clearly from where comes this 80 port, could you please say that container port will be take by default?